packages/slice-machine/src/features/onboarding/OnboardingProgressStepper.tsx

```diff
@@ -33,37 +35,46 @@ export const OnboardingProgressStepper = () => {
         isOpen={isDialogOpen}
         onClose={() => setDialogOpen(false)}
       />
-      <DropdownMenu>
+      <DropdownMenu open={isListOpen} onOpenChange={setListOpen}>
         <DropdownMenuTrigger>
           <Button
             color="grey"
             sx={{ width: "100%" }}
             renderEndIcon={EndCtaIcon}
+            onMouseEnter={() => setListOpen(true)}
           >
             {completedStepCount > 0 ? "Continue" : "Start"}
           </Button>
         </DropdownMenuTrigger>
-        <DropdownMenuContent align="start">
-          <DropdownMenuLabel>Progress</DropdownMenuLabel>
-          {steps.map((step) => {
-            const isCompleted = isStepComplete(step.id);
+        <DropdownMenuContent align="center" sideOffset={-56} minWidth={256}>
+          <div
+            onMouseLeave={() => {
+              setListOpen(false);
+            }}
+          >
+            <DropdownMenuLabel>
+              <Text variant="small" color="grey11">
+                Progress
+              </Text>
```

Comment:
This label should needs some margin/padding to be aligned with the content

![Screenshot 2024-07-25 at 16 14 06](https://github.com/user-attachments/assets/a75facca-1512-40ef-ae67-3e48dfd51cd4)

Reply:
Yes, I'm aware of that, this problem was present already in storybook of DropdownMenu. 
I showed this to Benjamin and he is fine with present version.

Reply:
You can easily apply it to this `Text`, why not make it pixel perfect? To me it looks misaligned 😅

Reply:
let me just double-check if I can do that with `Text`, maybe I'm missing something here

Reply:
Ok, fixed :)